scrapers/crec_scrapers/crec_detail_scraper.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import json
+import time
+
+def url(path):
+    return (
+        'https://www.govinfo.gov/wssearch/rb/crpt/'
+        + str(path)
+        + '/?fetchChildrenOnly=1'
+    )
+    
+def get_response_in_json(path):
+    response = requests.get(url(path))
+    return json.loads(response.text)
+
+def get_detail_data(url):
+    crec_data = {}
+    time.sleep(0.0)
+    response = requests.get(url)
+
+    data = json.loads(response.text)
+    #print('-----------', data.keys())
+    crec_data['title'] = data['title']
+    crec_data['pdf_link'] = data['download']['pdflink']
+    for col in data['metadata']['columnnamevalueset']:
```

Comment:
I think you need to create the `congress` values, too. Also check that you are creating the correct link to the bill number.

Reply:
Congress is not readily given in the scraped data. I am calculating it based on date in https://github.com/aih/FlatGov/blob/feature/committee_doc_main/server_py/flatgov/common/crec_data.py. Should I add it here as well?

Tested the bill number vs documents. Seems to be correct.

Reply:
No, I didn't see that you had it in `crec_data.py`. I just wanted to make sure you had the right version of the scraper that includes the congress info. Resolving this comment/conversation.